### DISCUSSION TITLE
API Swagger Payouts/batch

### DISCUSSION THREAD

**Discussion by skeutmeier** [MrNaif2018/bitcart#349] [Q&A] [answered]
Hi,

I am able to add payouts through the API, but I need to Approve and send them too.

Comparing with the backend actions, I assume that this can be achieved through the /api/payouts/batch endpoint.

I cannot see any documentation about the payload of that request (besides the IDs).

Can someone point me to the "Commands" available?
How can I batch the payouts into 1 transaction? Is there a undoczumented flag that I can set?

Thank you

![Screenshot from 2023-04-01 21-30-22](https://user-images.githubusercontent.com/73367887/229310553-1d3988d9-4c94-4b47-bba2-9d20fc2c6d23.png)
![Screenshot from 2023-04-01 21-30-14](https://user-images.githubusercontent.com/73367887/229310555-74c85065-fff3-42ba-b67a-d10844957402.png)

**Comment by MrNaif2018** [comment, marked as answer]
command is approve for example
https://github.com/bitcartcc/bitcart/blob/master/api/views/payouts.py#L20-L23

**Comment by skeutmeier** [comment]
Ok, Thank you. Got that working right now.
But every payout is its own TX. How can I bundle them into 1 TX?

I saw there may be a special HTTP Method, batch_action instead of POST.
But that didnt work.

Any hints on that?

Thank you

**Comment by MrNaif2018** [comment]
Pass batch: True in options dict for send batch command to batch it in one tx
Supported for BTC-based coins for now only
